src/routes/plans/[id]/+page.ts

```diff
@@ -21,8 +21,7 @@ export const load: PageLoad = async ({ parent, params, url }) => {
         throw redirect(302, `${base}/plans/${id}/merge`);
       }
       const initialActivityTypes = await effects.getActivityTypes(initialPlan.model_id);
-      const initialResourceTypes = await effects.getResourceTypes(initialPlan.model_id);
```

Comment:
Since resource types is being removed here this means we'll need that resource types table before merging in #315 which is slated for 1.1.0 if that's still the goal? Or we could bump that PR. Just curious about the timing.

Reply:
Yeah good point. I removed this here because for the Clipper model it makes the plan page load unreasonably long. I did not remove the `getResourceTypes` request tho since I know you need to use it. For your PR can we just fetch it on the plan page itself? I *think* it should be fine as long as we are not blocking the page load. The longer-term goal would be to cache the resource types but I don't want that to block your PR since everyone is pretty busy right now with other tasks.